ecl/hql/hqlexpr.cpp

```diff
@@ -4637,8 +4637,11 @@ switch (op)
         break;
     case no_funcdef:
         {
-            assertex(queryType()->getTypeCode() == type_function && queryChild(1) && queryChild(1)->getOperator() == no_sortlist);
+            IHqlExpression * formals = queryChild(1);
```

Comment:
This is improving the internal consistency checks to catch the problem sooner (in debug mode)

Reply:
Not sure why you say "in debug mode" here.

Reply:
Because this section of code is within a #ifdef VERIFY_EXPR_INTEGRITY, which is only generally enabled in debug.